package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "brfs",
-  "version": "1.4.3",
+  "version": "1.4.4",
```

Comment:
Can you revert this line? otherwise LGTM

Reply:
done